### ISSUE TITLE
Bridge Console Error Logging Exception

### ISSUE THREAD

**Issue by rufio620** [bridgedotnet/Bridge#2663]
There seems to be a bug in the bridge console error logging where the internal error method is not defined. Overall the bridge console is too invasive and should have a way to turn it off completely. The work around mentioned in the forums to replace calls using a startup method cause messages to be double output. It also appears the console wires up a global window error handler. In the case of this issue this causes the wrong exception message to be output to the browser console.

### Steps To Reproduce
 https://deck.net/c1604e2191d60f6b8d70ad15a3e3939c

See attached repro project. 
[Bridge16Test.zip](https://github.com/bridgedotnet/Bridge/files/988367/Bridge16Test.zip)

1. Compile and open the demo.html
2. Execute Bridge16Test.Bridge16Test.DoSomething() from the browser console.

### Expected Result

Error message should be printed to the console.

### Actual Result

Exception is thrown.
```
Uncaught ReferenceError: error is not defined
    at Function.con.error.Bridge.global.console.error (bridge.console.min.js:8)
    at Function.Bridge16Test.Bridge16Test.DoSomething (Bridge16Test.js:17)
    at <anonymous>:1:27
```

### See Also
* https://forums.bridge.net/forum/bridge-net-pro/bugs/4161-open-2663-console-error-fails

**Comment by rufio620** [comment]
After reviewing this a little more it appears the issues is in line 74 of birdge.console.js. The global console.error method is being overwritten. 

```javascript
if (con && con.error) {
                        Bridge.global.console.error = function (msg) {
                            error(msg);
                            Bridge.Console.error(msg);
                        }
                    }
```

**Comment by geoffreymcgill** [comment]
If you are using Bridge 16.0.0-beta or above, you can _"turn the Consol off"_ by just removing the **bridge.console.js** file from your .html page.

Can you share a Deck demonstrating the `Uncaught ReferenceError: error is not defined` you noted above?

Hope this helps.

**Comment by rufio620** [comment]
Repro is here https://deck.net/eaf7e5c601a0baeae7ab686c176f9951
It appears that it only occurs when overwriting the Bridge.Console.error function. Again probably stemming from the fact that the init is overwriting the native console.error.

Turning off the console by removing bridge.console.js does not seem to be possible when combining scripts via the "combineScripts": true setting in the bridge.json file.

Ideally there should be some option to allow the original console.error to be maintained as well as prevent automatic wiring up of the window.error event listener.

**Comment by geoffreymcgill** [comment]
If you're combining scripts, you can explicitly exclude **bridge.console.js** using the following `resources` config:

https://github.com/bridgedotnet/Bridge/issues/1994#issuecomment-296754941

The overrides you're using are not supported, and we do not recommend using.

If you want to prevent the automatic wiring up of the window.error event listener, just remove **bridge.console.js**.

Hope this helps.

**Comment by rufio620** [comment]
Yes that helps greatly. I was able to get the desired behavior using this solution coupled with the following override to ensure most calls to Bridge.Util.Console don't result in an error. Thanks for your help. 

`Script.Write("Bridge.Console = console");`

**Comment by geoffreymcgill** [comment]
Looks like there is a defect here that we didn't see on the first pass. We are investigating.

**Comment by geoffreymcgill** [comment]
A PR has been submitted with a fix. Once testing is complete, we will merge into the **master** branch and the fix will be available with the next public release.
